src/main/java/io/mosip/mimoto/controller/CredentialsController.java

```diff
@@ -56,12 +56,13 @@ public ResponseEntity<?> downloadCredentialAsPDF(@RequestParam Map<String, Strin
         try {
             String issuerId = params.get("issuer");
             String credentialType = params.get("credential");
+            String credentialValidity = params.get("vcExpiryTimes");
```

Comment:
this is not vcExpiry, this is vc storage expiry limit.
let's use attribute name as `vcStorageExpiryLimit`